recipes/autocnet/meta.yaml

```diff
@@ -0,0 +1,96 @@
+{% set name = "autocnet" %}
+{% set version = "1.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/DOI-USGS/Autocnet/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 233ec9b333305dbeb105a612a809b5cf8a4391c84d402ea92fe2615ba1cc290b
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  noarch: python
+  number: 0
+
+requirements:
+    build:
+      - python >=3.10
+      - csmapi
+      - dill
+      - geoalchemy2
+      - geopandas
+      - kalasiris
+      - knoten
+      - networkx
+      - opencv
+      - plio
+      - plurmy
+      - pyyaml
+      - redis-py
+      - scikit-image
+      - scipy
+      - shapely
+      - sqlalchemy-utils
+    host:
+      - python >=3.10
+      - setuptools
+    run:
+      - python >=3.10
+      - csmapi
+      - cython
+      - cyvlfeat
+      - matplotlib-base
+      - dill
+      - conda-forge::gdal
+      - geoalchemy2
+      - geopandas
+      - hoggorm
+      - imageio
+      - ipykernel
+      - kalasiris
+      - conda-forge::knoten
```

Comment:
In conda-forge, all packages are from conda-forge exclusively, no need this this syntax.